conf/nginx.conf

```diff
@@ -39,9 +57,6 @@ location __PATH__/ {
 		return 403;
 	}
 
-	rewrite ^__PATH__/play/ssid/(\w+)/type/(\w+)/oid/([0-9]+)/uid/([0-9]+)/name/(.*)$ __PATH__/play/index.php?ssid=$1&type=$2&oid=$3&uid=$4&name=$5 last;
-	rewrite ^__PATH__/play/ssid/(\w+)/type/(\w+)/oid/([0-9]+)/uid/([0-9]+)/client/(.*)/noscrobble/([0-1])/name/(.*)$ __PATH__/play/index.php?ssid=$1&type=$2&oid=$3&uid=$4&client=$5&noscrobble=$6&name=$7 last;
-
 	location __PATH__/rest {
```

Comment:
We can not login to subsonic api, if /rest is blocked. Should removing this be security issue?